hooks/post/30-upgrade.rb

```diff
@@ -36,6 +36,10 @@ def elasticsearch_message
   Kafo::Helpers.log_and_say :info, message
 end
 
+def add_export_distributor
+  Kafo::Helpers.execute('foreman-rake katello:upgrades:3.0:add_export_distributor')
+end
```

Comment:
I originally added this when I was pretty new to the team, thanks for catching this :panda_face: